articles/connecting-library-application-to-sql-server-using-windows-forms/index.md

```diff
@@ -0,0 +1,232 @@
+# Connecting-library-application-to-SQL-Server-using-Windows-forms
+
+### Introduction
+
+windows forms is one of the best tools used in making applications.
+With windows forms, you can make a variety of applications, from a mini calculator to a library application that connects to a database. In this tutorial, we will connect a library application using Windows forms to a database using SQL Server.
+
+In this tutorial, we will use a windows forms project and work on it, we will add new forms that will add edit & delete books from the library, we will also manage the authors, the stores & the publishers stored in the database. We will display the information of each attribute & refresh it each time a change accrues.
+
+### Prerequisites
+
+Before we begin, it would help you as the reader to have the following:
+
+- A basic understanding of C# programming language.
+- A basic understanding of Windows forms.
+- Visual Studio installed on your system.
+- SQL Server installed on your system.
+
+If you don’t have Visual Studio installed on your computer, you can check this article on how to set up the C# environment in Visual Studio [here](https://www.geeksforgeeks.org/setting-environment-c-sharp/), and for SQL Server you can download it from [here](https://www.microsoft.com/en-us/sql-server/sql-server-downloads). If you are new to Windows forms you can check this tutorial that would help you understand The basics of it [here](https://www.section.io/engineering-education/getting-started-with-windows-forms-using-c-sharp/).

```

Comment:
```suggestion
#### Installation guide
- To install Visual Studio and setup the work environment, refer [this](https://www.geeksforgeeks.org/setting-environment-c-sharp/) article on how to set up the C# environment in Visual Studio.
- To install SQL Server, you can download it from [here](https://www.microsoft.com/en-us/sql-server/sql-server-downloads).
- If you are new to Windows forms, you can check [this](/engineering-education/getting-started-with-windows-forms-using-c-sharp/) tutorial which would help you understand the basic concepts.
```

Reply:
Do not clutter too many information as a paragraph. Instead, list them out as points.

Also, while referring any other Section article, you need not type out the full URL. You can use `/engineering-education/getting-started-with-windows-forms-using-c-sharp/`